docs/examples/policies/chainloop-commit.yaml

```diff
@@ -17,18 +17,27 @@ kind: Policy
 metadata:
   name: chainloop-commit
 spec:
-  type: ATTESTATION
-  embedded: |
-    package main
+  policies:
+    - type: ATTESTATION
+      embedded: |
+        package main
```

Comment:
should we use the actual template we use with comments and so on?

Reply:
Yes, it makes sense. I'll add those comments